plugins/intel-usb4/meson.build

```diff
@@ -0,0 +1,30 @@
+if gusb.found()
```

Comment:
I wonder if we want to rename this meson option now:
`option('plugin_thunderbolt', type : 'feature', description : 'Thunderbolt support', deprecated: {'true': 'enabled', 'false': 'disabled'})
`

Reply:
https://github.com/fwupd/fwupd/pull/4804